tests/end2end/User-test.js

```diff
@@ -0,0 +1,182 @@
+import { E2EGlobal } from './helpers/E2EGlobal'
+import { E2EApp } from './helpers/E2EApp'
+import { E2EUser } from './helpers/E2EUser'
+
+
+describe('User Profile/Password editing', function () {
+
+    before("reload page and reset app", function () {
+        E2EApp.resetMyApp(true);
+        E2EApp.launchApp();
+    });
+
+    beforeEach("goto start page and make sure test user is logged in", function () {
+        E2EApp.gotoStartPage();
+        expect(E2EApp.isLoggedIn()).to.be.true;
+    });
+
+
+    it('Buttons Change Password and Edit Profile are not visible for an LDAP user', function () {
+        E2EApp.logoutUser();
+        expect(E2EApp.isLoggedIn()).to.be.false;
+
+        E2EApp.loginLdapUserWithCredentials('ldapUser1', 'ldapPwd1', false);
+        expect(E2EApp.isLoggedIn()).to.be.true;
+
+        if (E2EApp.isLoggedIn()) {
+            browser.click('#navbar-usermenu');
+            E2EGlobal.waitSomeTime();
+            expect(browser.isVisible('#navbar-dlgChangePassword')).to.be.false;
+            expect(browser.isVisible('#navbar-dlgEditProfile')).to.be.false;
+        }
+        browser.click('#navbar-usermenu');
+
+        E2EApp.logoutUser();
+        expect(E2EApp.isLoggedIn()).to.be.false;
+        E2EApp.loginUser();
+    });
+
+    it('User can successefully change his password', function () {
+        if (E2EApp.isLoggedIn()) {
```

Comment:
If someone inserts a test before this test that leaves the user logged out, this test will do nothing but be "green". You can euther ensure loged in user or you can let the test fail, of noone is logged in...